balrogscript/requirements/base.in

```diff
@@ -1,4 +1,4 @@
-balrogclient>=0.2.0
+balrogclient>=1.0.0
```

Comment:
Let's bump this to 1.1.0 -- 1.0.0 still enforced its own dependency versions, which can cause some trouble when we deploy.

Reply:
+1